include/zephyr/usb/class/usb_hub.h

```diff
@@ -12,33 +12,41 @@
 #ifndef ZEPHYR_INCLUDE_USB_CLASS_USB_HUB_H_
 #define ZEPHYR_INCLUDE_USB_CLASS_USB_HUB_H_
 
-/** USB Hub Class Feature Selectors defined in spec. Table 11-17 */
-#define USB_HCFS_C_HUB_LOCAL_POWER	0x00
-#define USB_HCFS_C_HUB_OVER_CURRENT	0x01
-#define USB_HCFS_PORT_CONNECTION	0x00
-#define USB_HCFS_PORT_ENABLE		0x01
-#define USB_HCFS_PORT_SUSPEND		0x02
-#define USB_HCFS_PORT_OVER_CURRENT	0x03
-#define USB_HCFS_PORT_RESET		0x04
-#define USB_HCFS_PORT_POWER		0x08
-#define USB_HCFS_PORT_LOW_SPEED		0x09
-#define USB_HCFS_C_PORT_CONNECTION	0x10
-#define USB_HCFS_C_PORT_ENABLE		0x11
-#define USB_HCFS_C_PORT_SUSPEND		0x12
-#define USB_HCFS_C_PORT_OVER_CURRENT	0x13
-#define USB_HCFS_C_PORT_RESET		0x14
-#define USB_HCFS_PORT_TEST		0x15
-#define USB_HCFS_PORT_INDICATOR		0x16
+/**
+ * @name USB Hub Class Feature Selectors defined in specification, Table 11-17.
+ * @{
+ */
+#define USB_HCFS_C_HUB_LOCAL_POWER	0x00    /**< Local Power Source */
+#define USB_HCFS_C_HUB_OVER_CURRENT	0x01    /**< Over-current */
+#define USB_HCFS_PORT_CONNECTION	0x00    /**< Current Connect Status */
+#define USB_HCFS_PORT_ENABLE		0x01    /**< Port Enabled/Disabled */
+#define USB_HCFS_PORT_SUSPEND		0x02    /**< Suspended */
+#define USB_HCFS_PORT_OVER_CURRENT	0x03    /**< Over-current */
+#define USB_HCFS_PORT_RESET		0x04    /**< Reset */
+#define USB_HCFS_PORT_POWER		0x08    /**< Port Power */
+#define USB_HCFS_PORT_LOW_SPEED		0x09    /**< Low-speed device attached */
+#define USB_HCFS_C_PORT_CONNECTION	0x10    /**< Connect Status Change */
+#define USB_HCFS_C_PORT_ENABLE		0x11    /**< Port Enable/Disable Change */
+#define USB_HCFS_C_PORT_SUSPEND		0x12    /**< Suspended Change */
+#define USB_HCFS_C_PORT_OVER_CURRENT	0x13    /**< Over-current Indicator Change */
+#define USB_HCFS_C_PORT_RESET		0x14    /**< Reset Change */
+#define USB_HCFS_PORT_TEST		0x15    /**< Port Test Mode */
+#define USB_HCFS_PORT_INDICATOR		0x16    /**< Port Indicator Control */
```

Comment:
These comments are completely unnecessary, the names are self-explanatory.  This should be rather internally hidden.

Reply:
There is no official "short description" for these as the specification just uses the identifiers (without the `USB_HCFS_` prefix) directly.

Reply:
> There is no official "short description" for these as the specification just uses the identifiers (without the `USB_HCFS_` prefix) directly.

I actually took these verbatim from the specification, as they are "spelled out" in 11.24.2.7.1 and 11.24.2.7.2.

Reply:
OK, I see. You took the bold text as description.

Reply:
@jfischer-no see comments from @kartben